pkgs/applications/misc/ulauncher/default.nix

```diff
@@ -42,18 +44,19 @@ python27Packages.buildPythonApplication rec  {
     wmctrl
   ];
 
-  propagatedBuildInputs = with python27Packages; [
+  propagatedBuildInputs = with python36Packages; [
+    mock
+    mypy
     dbus-python
-    notify
     pygobject3
     pyinotify
-    pysqlite
     python-Levenshtein
     pyxdg
     websocket_client
+    librsvg
   ];
```

Comment:
I believe `librsvg` should be in `buildInputs` because it's not a python dependency.

I also believe you're missing the following dependencies
* `mypy_extensions` https://github.com/Ulauncher/Ulauncher/blob/5.6.1/requirements.txt#L6
* it claims `websocket_client` https://github.com/Ulauncher/Ulauncher/blob/5.6.1/docs/extensions/libs.rst